frontend/src/features/admin-form/create/featureTour/FeatureTourTooltip.tsx

```diff
@@ -0,0 +1,84 @@
+import { BiBulb, BiRightArrowAlt } from 'react-icons/bi'
+import { Box, BoxProps, CloseButton, Flex, Icon, Text } from '@chakra-ui/react'
+
+import Badge from '~components/Badge'
+import Button from '~components/Button'
+
+interface FeatureTourButtonProps {
+  onClick: (e: React.MouseEvent<HTMLElement>) => void
+}
+
+interface FeatureTourStep {
+  content: React.ReactNode
+  title?: React.ReactNode
+}
+
+interface FeatureTourTooltipProps {
+  step: FeatureTourStep
+  tooltipProps: BoxProps
+  primaryProps: FeatureTourButtonProps
```

Comment:
nit: don't actually need a specific prop for this, you can also use ButtonProps, either ours or ChakraUI's